appimage/Readme.md

```diff
@@ -0,0 +1,24 @@
+AppImage Packaging
+==================
+
+Creates an AppImage that contains `pktvisord` `pktvisor-pcap` and `pktvisor-cli`. This implementation builds the AppImage in a docker using the assets generated by the docker [build](../docker). Because not all of the assets are statically complied we use [Exodus](https://github.com/intoli/exodus) to capture a minimal set of decencies for the AppImage 
```

Comment:
few typos: s/complied/compiled but maybe better word is "linked", s/decencies/dependencies